quora-service/src/main/java/com/upgrad/quora/service/dao/AnswerDao.java

```diff
@@ -19,11 +21,42 @@ public AnswerEntity createAnswer(AnswerEntity answerEntity) {
         return answerEntity;
     }
 
+    public AnswerEntity getAnswerByUuId(final String uuid) {
+        try {
+            return entityManager.createNamedQuery("AnswerFromUuid", AnswerEntity.class).setParameter("answerUuid", uuid).getSingleResult();
+        } catch (NoResultException nre) {
+            return null;
+        }
+    }
+
+    public Boolean isUserOwnerOfAnswer(String answerUuid, String userUuid) {
+
+        try {
+            AnswerEntity answerEntity = entityManager.createNamedQuery("validateOwnership", AnswerEntity.class).setParameter("answerUuid", answerUuid).setParameter("userUuid", userUuid).getSingleResult();
+            if (answerEntity == null) {
+                return false;
+            }
+            else {
+                return true;
+            }
+        }catch (NoResultException nre)
+        {
+            return false;
+        }
+    }
+
+    public AnswerEntity deleteAnswer(AnswerEntity answerEntity)
```

Comment:
This operation is transactional. Please introduce.

Reply:
Added in service class